<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerEMSegmentSpecifyIntensityDistributionStep</class>
 <widget class="qSlicerEMSegmentWorkflowWidgetStep" name="qSlicerEMSegmentSpecifyIntensityDistributionStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="AnatomicalTreeGroupBox">
     <property name="title">
      <string>Anatomical Tree</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qSlicerEMSegmentAnatomicalTreeWidget" name="EMSegmentAnatomicalTreeWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="IntensityDistributionTab">
      <attribute name="title">
       <string>Intensity Distribution</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="DistributionSpecificationMethodLabel">
         <property name="text">
          <string>Specification:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="DistributionSpecificationMethodComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="MeanLabel">
         <property name="text">
          <string>Mean:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ctkMatrixWidget" name="MeanMatrixWidget">
         <property name="toolTip">
          <string>Double-click to set the intensity distribution mean.</string>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CovarianceLabel">
         <property name="text">
          <string>Covariance:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ctkMatrixWidget" name="CovarianceMatrixWidget">
         <property name="toolTip">
          <string>Double-click to set the intensity distribution covariance.</string>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PlotDistributionButton">
       <property name="text">
        <string>Plot Distributions</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkWorkflowWidgetStep</class>
   <extends>QWidget</extends>
   <header>ctkWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentAnatomicalTreeWidget</class>
   <extends>qSlicerEMSegmentWidget</extends>
   <header>qSlicerEMSegmentAnatomicalTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerEMSegmentWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWorkflowWidgetStep</class>
   <extends>ctkWorkflowWidgetStep</extends>
   <header>qSlicerEMSegmentWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerEMSegmentSpecifyIntensityDistributionStep</sender>
   <signal>mrmlManagerChanged(vtkEMSegmentMRMLManager*)</signal>
   <receiver>EMSegmentAnatomicalTreeWidget</receiver>
   <slot>setMRMLManager(vtkEMSegmentMRMLManager*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerEMSegmentSpecifyIntensityDistributionStep</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>EMSegmentAnatomicalTreeWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
